### PR TITLE
Move the clipboard to its own widget

### PR DESCRIPTION
**Pull request by piuccio**
And fix the copy paste on the clipboard

### CODE REVIEW

File: facia-tool/public/css/style.css [hunk 1]
```diff
@@ -1136,6 +1136,10 @@ button {
     max-height: 75%;
 }
 
+.clipboard-only .cilpboard-container {
```
Comment (robertberry) [inline facia-tool/public/css/style.css]: `cilpboard-container`?

File: facia-tool/public/js/widgets/clipboard.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<div class="col__inner cilpboard-container">
```
Comment (robertberry) [inline facia-tool/public/js/widgets/clipboard.html]: Looks like you made the same mistake here conveniently ...

**Comment by robertberry** [comment]
Otherwise :+1:

File: facia-tool/public/js/widgets/clipboard.html (hunk 2 above)
Comment (piuccio) [inline facia-tool/public/js/widgets/clipboard.html]: :) thanks for spotting it, i'll fix

**Pull request merged**